### DISCUSSION TITLE
How to export multiple types of modules at the same time.

### DISCUSSION THREAD

**Discussion by Lydanne** [swc-project/swc#4324] [Q&A]
I want to export both CJS and ESM modules at the same time, but I can't find a method.

**Comment by kwonoj** [comment]
we don't emit multiple outputs at the same time for now. You may need run transform twice.

**Reply by Lydanne** [reply]
It's okay to run twice. Is there such an example?

**Reply by Lydanne** [reply]
I'm using `spack` and `spack.config.js`, but I found that some configurations(eg: `options`) don't work.
